### ISSUE TITLE
oraclejdk is broken

### ISSUE THREAD

**Issue by yawnt** [NixOS/nixpkgs#15493]
## Issue description

I'm getting a No such file or directory when trying to build OracleJDK8

```
mv: cannot stat '/nix/store/9gawkw84h4j0jk0bp37l4fm13mnvnbn7-oraclejre-8u91/bin/jmc': No such file or directory
```

I checked in `$JDK_ARCHIVE/jre/bin` and jmc is not present, so I'm not sure why it should be trying to copy it

### Steps to reproduce

Add `oraclejdk8` to `environment.systemPackages` and build

## Technical details

* System: (NixOS: `nixos-version`, Ubuntu/Fedora: `lsb_release -a`, ...)
* Nix version: (run `nix-env --version`)
* Nixpkgs version: (run `nix-instantiate --eval '<nixpkgs>' -A lib.nixpkgsVersion`)

**Comment by boobiesinc** [comment]
>Oracle, which acquired Java when it purchased Sun Microsystems in 2010, says Google infringed its copyrights by using 37 Java APIs in Android. In 2012, a judge ruled that APIs can't be copyrighted at all, but an appeals court disagreed. Now Oracle may seek up to $9 billion in damages, while Google is arguing that its use of the 37 APIs constitutes "fair use."

http://arstechnica.com/tech-policy/2016/05/at-trial-top-android-coder-explains-oracles-questions-on-scrubbed-source-code/

**Comment by fpletz** [comment]
Are you on NixOS 16.03 or 16.09 (master)? This should be fixed on master as of d51056383731b41a027778249e356bacc46a9385 but isn't probably not yet released in the unstable channel.

**Comment by fpletz** [comment]
Just checked, the failing code isn't in 16.03, so this is fixed by the mentioned commit above and will be available in the unstable channel soon.
